.github/ci_settings.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+#  This file and its contents are licensed under the Apache License 2.0.
+#  Please see the included NOTICE for copyright information and
+#  LICENSE-APACHE for a copy of the license.
+
+# Common settings for our CI jobs
+
+PG12_EARLIEST = "12.0"
+PG12_LATEST = "12.11"
+PG13_EARLIEST = "13.2"
+PG13_LATEST = "13.7"
+PG14_EARLIEST = "14.0"
+PG14_LATEST = "14.4"
```

Comment:
What about get the LATEST versions from here: https://www.postgresql.org/versions.rss ?

Reply:
We want this to be an explicit decission and not depend on external services. Also on release the new version is not available on all platforms so that would mean every new postgres release would break our CI till every platform has all the packages.